README.rst

```diff
@@ -270,6 +270,21 @@ Config settings
                 root /www/resource_cache;
             }
 
+6.  If you are not serving your archived resources through a webserver, you can upload them to ``s3filestore``. 
+    Make sure you have properly setup `ckanext-s3filestore <https://github.com/datopian/ckanext-s3filestore>`_ for this.
+
+    Add the settings to the CKAN config file:
+
+      * ``ckanext.archiver.s3upload_enable`` = ``True`` to enable upload to filestore. If unset or ``False``, the resources are going to be archived locally instead of on s3filestore.
+      * ``ckanext.s3filestore.aws_storage_path`` = ``my-site-name``. Your filestore project path. For example ckan/storage_path/archived_resource_dir. This must be set in order to upload files to filestore.
+
+    The resources are uploaded to s3filestore in the directory ``s3filestore.aws_bucket_name/s3filestore.aws_storage_path/archived_resources/resource_id/``.
```

Comment:
uploaded to cloud storage or S3 file storage (s3filestore is an extension name)

Reply:
@zelima 

Everything in the code is done keeping s3filestore in mind

https://github.com/datopian/ckanext-archiver/blob/feature/filestore_resource_upload/ckanext/archiver/tasks.py#L565-L599

Would other cloud-storages be compatible with this?

Reply:
@MuhammadIsmailShahzad I don't know, will it?

Reply:
Speaking of that, don't we need updating requirements.txt?

Reply:
It is only going to work for s3filestore. So not for all cloudstorages.

And we are adding no new packages or anything. Everything needed for filestore is being installed by s3filestore extension.